### PR TITLE
Patterns_Problems fixes#1367

### LINKED ISSUE #1367: Patterns_problems
<hr>

## Description 📜
This documentation will contain  detailed explanation of Patterns problems using C/C++.
* Number pattern .
* Star Pattern

... etc...

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

### PR DESCRIPTION
**Pull request by pilipi-puu-puu**
<hr>

## Description 📜

Added several pattern problems using C.
Fixes #1367 

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by pilipi-puu-puu** [comment]
@Saviour1001 kindly review this.

File: C_CPP/Pattern/README.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+   printf(" ");
+   }
+   printf("\n"); 
+ }
+}
+```
+### Hope these problems will help you✨.
+## Thank You😁
+
+  -Created by *[Anushka Paul](https://github.com/pilipi-puu-puu)*
```
Comment (deeqakkk) [inline C_CPP/Pattern/README.md:265]: anushka, mentioning your name or id is not allowed. Kindly remove this.
rest of things are good, you can mention some comments as well for easy understanding.

**Review by deeqakkk — CHANGES_REQUESTED** [review]

File: C_CPP/Pattern/README.md (hunk 1 above)
Comment (pilipi-puu-puu) [reply C_CPP/Pattern/README.md:265]: @deeqakkk ok

**Comment by pilipi-puu-puu** [comment]
@deeqakkk I have removed my id Sir. Kindly review it😊🙏

**Comment by Saviour1001** [comment]
You don't need to create a new folder for this. Could you add it in the Algorithmic Approaches folder [here](https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Algorithmic%20Approaches).

**Comment by pilipi-puu-puu** [comment]
@Saviour1001 It will be great if you kindly elaborate me how to add in Algorithm approaches folder🙏.
Like in that folder I need to upload my whole Pattern folder[README.md] , then again I have to make a PR?

**Comment by Saviour1001** [comment]
Ping me on discord

**Comment by pilipi-puu-puu** [comment]
@deeqakkk is it ok now to get merged? then I can work on other issues😊

**Comment by Saviour1001** [comment]
image
Could you place this file back ,

**Comment by pilipi-puu-puu** [comment]
@Saviour1001 sorry I can't .

**Comment by Suvraneel** [comment]
@pilipi-puu-puu  Its easy just copy contents from gwoc repo that directory & paste in your branch...

Theoretically just restoring the deleted file

**Comment by pilipi-puu-puu** [comment]
@Suvraneel is it ok now?🥺

**Comment by Suvraneel** [comment]
@pilipi-puu-puu  Actually no... Your branch is backdated (follows old roadmap of C/CPP branch)
Just save your work...

(Update your repo by fetch n merge or by deleting & forking again)

**Pull request closed without merging**